README.md

```diff
@@ -0,0 +1,96 @@
+# Net Agent
+
+[![Crates.io](https://img.shields.io/crates/v/net-agent.svg)](https://crates.io/crates/net-agent)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+`net-agent` is a binary crate that allows you to capture network traffic and load it to [my.netstalker.io](https://my.netstalker.io) for further analysis.
+
+## Installation
+
+To install `net-agent`, use Cargo:
+
+```shell
+cargo install net-agent
+```
+
+## Requirements
+Before running `net-agent`, make sure you have the necessary dependencies installed for your operating system:
+
+### Linux
+
+- Ubuntu/Debian:
+    ```shell
+    sudo apt-get install libpcap-dev
+    ```
+
+- Fedora:
+    ```shell
+    sudo dnf install libpcap-devel
+    ```
+- Arch:
+    ```shell
+    sudo pacman -S libpcap
+    ```
+### macOS
+
+- Install Xcode Command Line Tools::
+    ```shell
+    xcode-select --install
+    ```
+- Install libpcap using Homebrew::
+    ```shell
+    brew install libpcap
+    ```
+
```

Comment:
I would like you to be consistent with style. If you leave empty line somewhere, please think if we place such symbols (new line) in same circumstances at other places.